src/types.ts

```diff
@@ -59,13 +59,16 @@ export type PartialGameState = Pick<State, 'G' | 'ctx' | 'plugins'>;
 export type StageName = string;
 export type PlayerID = string;
 
-export type StageArg = StageName | { stage?: StageName; moveLimit?: number };
+export type StageArg =
+  | StageName
+  | { stage?: StageName; minMoves?: number; moveLimit?: number };
 
 export interface ActivePlayersArg {
   currentPlayer?: StageArg;
   others?: StageArg;
   all?: StageArg;
   value?: Record<PlayerID, StageArg>;
+  minMoves?: number;
   moveLimit?: number;
```

Comment:
We’ll need to update the types something like this. Using a `@deprecated` JSDoc tag for `moveLimit` adds some editor hinting where supported to further help highlight the change.

```suggestion
  minMoves?: number;
  maxMoves?: number;
  /** @deprecated Use `minMoves` and `maxMoves` instead. */
  moveLimit?: number;
```

Same would go for the `StageArg` type.